style.css

```diff
@@ -0,0 +1,60 @@
+main {
+    background-color: rgba(204, 204, 204, 0.4);
+    padding-top: 20px;
+    font-family: "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif;
+}
+
+h1 {
+    text-align: center;
+    font-size: 40px;
+}
+
+h1+p {
```

Comment:
Could it be a common rule for all possible pages with h1? Or just for this case? The answer to these questions will reveal a solution: h1 + p or class.